vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/VariantExporterControllerTest.java

```diff
@@ -335,6 +336,26 @@ public void testFilterUsingIntersectingRegions() throws Exception {
         checkOrderInOutputFile(outputFile);
     }
 
+    @Test
+    public void testDivideChromosomeInChunks() throws Exception {
+        String studyId = "7";
+        List<String> studies = Collections.singletonList(studyId);
+        MultivaluedMap<String, String> queryParameters = new MultivaluedHashMap<>();
+        int blockSize = Integer.parseInt(evaTestProperties.getProperty("eva.htsget.blocksize"));
+        VariantExporterController controller = new VariantExporterController(
+                TestDBRule.getTemporaryDBName(TestDBRule.HUMAN_TEST_DB),
+                studies, evaTestProperties, queryParameters, blockSize);
+        List<Region> regions = controller.divideChromosomeInChunks("1", 500, 1499);
```

Comment:
I will add a line jump before List<Region> regions for improving the readability of this code. (3 blocks: initialization, first test with asserts, second test with asserts)